nucypher/network/server.py

```diff
@@ -154,8 +154,9 @@ def condition_chains():
         """
         # TODO: When non-evm chains are supported, bump the version.
         #  this can return a list of chain names or other verifiable identifiers.
-
-        payload = {"version": 1.0, "evm": list(this_node.condition_providers)}
+        providers = this_node.condition_providers.providers
```

Comment:
probably dumb question, should it be `condition_provider_manager` instead of `condition_providers`?

Reply:
✔️  - Not dumb at all. In fact, great catch 🎸 . That was a bug from a rename, but the mocked integration test still passed. Fixed now.